src/Microsoft.Extensions.OData.Migration/MigrationOptions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Microsoft.Extensions.OData.Migration
+{
+    /// <summary>
+    /// Container for parameters to pass into middleware.
+    /// </summary>
+    public sealed class MigrationOptions
+    {
+        // Service root (e.g. the https://foo:80/odata part of https://foo:80/odata/Products) is required for parser
+        public Uri ServiceRoot { get; set; }

```

Comment:
If it's required, we probably should also pass it to UseMigrationMiddleware

Reply:
Removed ODataMigrationOptions entirely; service root and models are all now required parameters